packages/containers/sandbox/src/app/expressions/index.js

```diff
@@ -0,0 +1,9 @@
+import { selectors } from '@talend/react-cmf';
+
+function getUsersCollectionAsArray({ context }, collectionPath) {
+	const list = selectors.collections.get(context.store.getState(), collectionPath);
+
+	return list && list.toJS() ? list.toJS() : [];
+}
+
+export default { getUsersCollectionAsArray };
```

Comment:
This expression wraps CMF collection selector so that the props arrives as a JS array of JS objects instead of an immutable list of maps.